lib/tasks/locale.rake

```diff
@@ -208,22 +194,24 @@ namespace :locale do
       "https://raw.githubusercontent.com/ManageIQ/react-ui-components/#{checkout_branch}/locale/react-ui-components.pot"
```

Comment:
This extra pot can be dropped  (not necessarily for this PR).  We archived this repo a LONG time ago when the code was folded into the manageiq-ui-classic repo.

Reply:
Should this be where the service UI pot file gets sucked in?

Reply:
I guess it's not modular though so not sure how the translated strings will make it back to service ui.

Reply:
I'll do this after this is merged so I can get the related https://github.com/ManageIQ/manageiq/pull/22438 in so I can try to get adhoc github action for generating the json files from the ui classic repo.